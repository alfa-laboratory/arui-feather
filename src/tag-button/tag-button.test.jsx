/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import React from 'react';
import { mount } from 'enzyme';

import TagButton from './tag-button';

describe('tag-button', () => {
    it('renders without problems', () => {
        let tagButton = mount(<TagButton />);
        expect(tagButton).toMatchSnapshot();
    });
});
