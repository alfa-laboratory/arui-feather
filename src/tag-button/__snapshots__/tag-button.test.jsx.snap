// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tag-button renders without problems 1`] = `
<TagButton
  formNoValidate={false}
  size="m"
  tag="button"
  type="button"
>
  <button
    className="tag-button tag-button_size_m tag-button_theme_alfa-on-white"
    formNoValidate={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseOut={[Function]}
    onMouseUp={[Function]}
    role="button"
    type="button"
  />
</TagButton>
`;
